src/test_DisplayMenu.py

```diff
@@ -0,0 +1,59 @@
+import pytest
```

Comment:
Can you put the unit test code in the main test_indexGame.py instead

Reply:
Okay can I will move everything over bah